### PR TITLE
dev: restore autopep8

### PR DESCRIPTION
**Pull request by dcramer**

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -34,7 +34,7 @@
 
     "python.linting.pylintEnabled": false,
     "python.linting.flake8Enabled": true,
-    "python.formatting.provider": "yapf",
+    "python.formatting.provider": "autopep8",
     // https://github.com/DonJayamanne/pythonVSCode/issues/992
     "python.formatting.yapfArgs": ["--style", "${workspaceRoot}/setup.cfg"],
```
Comment (dcramer) [inline .vscode/settings.json]: i kept this setting but its unused

**Comment by getsentry-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          2 Warnings
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Changes to build requirements</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">You should update CHANGES due to the size of this PR</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">danger</a>
</p>

**Comment by dcramer** [comment]
if we want to do it, lets do it

yapf has proven problematic, and autopep8 gives us a LITTLE bit of standardization on newlines/parens

File: setup.py [hunk 2]
```diff
@@ -106,6 +110,7 @@
     'honcho>=0.7.0,<0.8.0',
     'kombu==3.0.35',
     'lxml>=3.4.1',
+
```
Comment (mattrobenolt) [inline setup.py]: why

Comment (dcramer) [inline setup.py]: yeah idk

**Pull request merged**